setup.py

```diff
@@ -4,6 +4,7 @@
 from codecs import open
 from os.path import abspath, dirname, join
 from subprocess import call
+import requests
```

Comment:
we actually don't need this here,   we need to insert it into line 60 e.g. install_requires = ['Click', 'prettytable', 'reqeuests'],